spark/src/main/scala/io/delta/sql/parser/DeltaSqlParser.scala

```diff
@@ -560,6 +560,19 @@ class DeltaSqlAstBuilder extends DeltaSqlBaseBaseVisitor[AnyRef] {
       ifExists = ctx.EXISTS != null)
   }
 
+  /**
+   * `ALTER TABLE ALTER COLUMN SYNC IDENTITY` command.
```

Comment:
It is also the command for `ALTER TABLE ... CHANGE COLUMN ... SYNC IDENTITY` right?